INSTALL.md

```diff
@@ -1,5 +1,5 @@
 # INSTALL
-How to install Pinkaes in a production-like environment. This guide assumes a basic knowledge about networking and systems administration. 
+How to install Pinkaes in a production-like environment. This guide assumes a basic knowledge about networking and systems administration.
```

Comment:
you could take advantage to fix that "Pinkaes"

Reply:
Good spot, updated.